.gitignore

```diff
@@ -63,6 +63,9 @@ target/
 # Sublime Text 2
 *.sublime*
 
+# Visual Studio Code
```

Comment:
Could you remove this from the PR since it isn't really relavant?

Hint: you can add this rule in your local .gitignore

Reply:
Sure.